modules/local/dream/differential/tests/main.nf.test.snap

```diff
@@ -0,0 +1,503 @@
+                            "reference": "mCherry",
+                            "target": "hND6",
+                            "blocking_factors": "sample_number"
+                        },
+                        "treatment_mCherry_hND6.dream.model.txt:md5,d2113d82b76046c319e6602da2ad74d6"
+                    ]
+                ],
+                "normalised_counts": [
+                    
+                ],
+                "rdata": [
+                    [
+                        {
+                            "id": "treatment_mCherry_hND6",
+                            "contrast_variable": "treatment",
+                            "reference": "mCherry",
+                            "target": "hND6",
+                            "blocking_factors": "sample_number"
+                        },
+                        "treatment_mCherry_hND6.MArrayMM.dream.rds:md5,9197daf020341f1fc92636c762cb40e0"
+                    ]
+                ],
+                "results": [
+                    [
+                        {
+                            "id": "treatment_mCherry_hND6",
+                            "contrast_variable": "treatment",
+                            "reference": "mCherry",
+                            "target": "hND6",
+                            "blocking_factors": "sample_number"
+                        },
+                        "treatment_mCherry_hND6.dream.results.tsv:md5,bd28cb586bb40ad9f0b7069cb3ada0a2"
+                    ]
+                ],
+                "session_info": [
+                    [
+                        {
+                            "id": "treatment_mCherry_hND6",
+                            "contrast_variable": "treatment",
+                            "reference": "mCherry",
+                            "target": "hND6",
+                            "blocking_factors": "sample_number"
+                        },
+                        "treatment_mCherry_hND6.R_sessionInfo.log:md5,e591353ef3f5d63a74889d57f7ce37c6"
+                    ]
+                ],
+                "versions": [
+                    "versions.yml:md5,56f3e82916abb64aa39ae9d008d71978"
+                ]
+            }
+        ],
+        "meta": {
+            "nf-test": "0.9.2",
+            "nextflow": "24.10.3"
+        },
+        "timestamp": "2024-12-27T18:45:31.211486573"
+    },
+    "Mus musculus - expression table - contrasts + blocking factors stub": {
+        "content": [
+            {
+                "0": [
+                    [
+                        {
+                            "id": "treatment_mCherry_hND6",
+                            "contrast_variable": "treatment",
+                            "reference": "mCherry",
+                            "target": "hND6",
+                            "blocking_factors": "sample_number"
+                        },
+                        "treatment_mCherry_hND6.dream.results.tsv:md5,d41d8cd98f00b204e9800998ecf8427e"
```

Comment:
Any file with an md5sum of "d41d8cd98f00b204e9800998ecf8427e" is empty, please review if this is expected

Reply:
@nschcolnicov This is part of the module's **stub** test

Reply:
My bad, its ok then!